src/renderer/components/Recent.tsx

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react';
+
+const RecentList = () => {
+  const defaultValue = ['No Recent Connections'];
+  const [recent, setRecent] = useState<string[]>(defaultValue);
+
+  useEffect(() => {
+    const fetchRecent = async () => {
+      const newRecent = await window.procedures.ipcRenderer.fetchDatabases();
```

Comment:
This should be showing recent Server Connections that we are storing in Sqlite, not a list of databases on the current server connection.
This will not work once our hardcoded value are removed since the connection menu only comes *after* the launcher screen so this list won't be populated.